[2024-10-17T21:48:36.311+0100] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-17T21:48:36.314+0100] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskBranch scheduled__2024-10-16T00:00:00+00:00 [queued]>
[2024-10-17T21:48:36.316+0100] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskBranch scheduled__2024-10-16T00:00:00+00:00 [queued]>
[2024-10-17T21:48:36.316+0100] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-17T21:48:36.319+0100] {taskinstance.py:2888} INFO - Executing <Task(BranchPythonOperator): taskBranch> on 2024-10-16 00:00:00+00:00
[2024-10-17T21:48:36.322+0100] {standard_task_runner.py:72} INFO - Started process 5532 to run task
[2024-10-17T21:48:36.325+0100] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'cron_catchup_backfill', 'taskBranch', 'scheduled__2024-10-16T00:00:00+00:00', '--job-id', '283', '--raw', '--subdir', 'DAGS_FOLDER/cron_catchup_backfill.py', '--cfg-path', '/var/folders/nl/1x330z2x7yzb268yb0bbvjcr0000gn/T/tmp0lng5cf3']
[2024-10-17T21:48:36.326+0100] {standard_task_runner.py:105} INFO - Job 283: Subtask taskBranch
[2024-10-17T21:48:36.338+0100] {task_command.py:467} INFO - Running <TaskInstance: cron_catchup_backfill.taskBranch scheduled__2024-10-16T00:00:00+00:00 [running]> on host mujtabas-macbook-pro.local
[2024-10-17T21:48:36.359+0100] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ali' AIRFLOW_CTX_DAG_ID='cron_catchup_backfill' AIRFLOW_CTX_TASK_ID='taskBranch' AIRFLOW_CTX_EXECUTION_DATE='2024-10-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-16T00:00:00+00:00'
[2024-10-17T21:48:36.359+0100] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-17T21:48:36.363+0100] {baseoperator.py:405} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-10-17T21:48:36.365+0100] {python.py:240} INFO - Done. Returned value was: taskD
[2024-10-17T21:48:36.365+0100] {branch.py:38} INFO - Branch into taskD
[2024-10-17T21:48:36.366+0100] {skipmixin.py:230} INFO - Following branch {'taskD'}
[2024-10-17T21:48:36.367+0100] {skipmixin.py:278} INFO - Skipping tasks [('taskE', -1), ('taskC', -1)]
[2024-10-17T21:48:36.372+0100] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-17T21:48:36.373+0100] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cron_catchup_backfill, task_id=taskBranch, run_id=scheduled__2024-10-16T00:00:00+00:00, execution_date=20241016T000000, start_date=20241017T204836, end_date=20241017T204836
[2024-10-17T21:48:36.386+0100] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-17T21:48:36.392+0100] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-17T21:48:36.394+0100] {local_task_job_runner.py:245} INFO - ::endgroup::
